src/main/java/com/fasterxml/jackson/databind/SerializerProvider.java

```diff
@@ -249,6 +249,7 @@ protected SerializerProvider(SerializerProvider src)
         _serializationView = null;
         _serializerFactory = null;
         _knownSerializers = null;
+        _dateFormat = null;
```

Comment:
Same here, I do not think it should be set here; it's a lazily initiated field for reuse, not set in any of constructors.